### ISSUE TITLE
JOB: Senior React Native Mobile Developer Web3 (Remote)

### ISSUE THREAD

**Issue by CarmelHR** [react-brasil/vagas#2721]
Please Note: This is a fully remote position.
APPLY HERE: https://apply.workable.com/buzz-3/j/32DDCB5885/

About Buzz
There is a steep learning curve when it comes to crypto. Only once overcome will we have mass adoption. Buzz Wallet is a mobile crypto wallet that aims to debunk the belief that decentralised finance is inherently complex. We will amend the frustratingly poor user experience and fragmentation that millions of newcomers, and crypto natives, face. We will seamlessly merge the centralised and decentralised world.
Check out Buzz Wallet: buzzwallet.io
Buzz Wallet falls under Aldrin Labs. Another product under Aldrin Labs is our DEX Aldrin (aldrin.com).
Aldrin Labs is fully remote company. We are interested in capturing the best talent, no matter where you live.

The Role
We are looking for a Senior React Native Mobile Developer with extensive experience in designing, building, and deploying highly secure, scalable, and maintainable client-facing UI systems.
You will be one of the senior developers of the mobile app working closely with our designers, product owners, and backend engineers to develop and maintain high-quality mobile applications for a global audience in an extremely fast-growing industry. We work with a modern tech-stack. We use Typescript, hooks, eslint, reanimated, automated CI and deployments, etc. We work with a design system and an updated Figma design.
If you are extremely passionate about mobile platforms and translating code into user-friendly apps, we would like to meet you. Ultimately, you should be able to help design and build the next generation of our mobile applications that will impact and shape the financial lives of a growing audience.

You will:
Contribute code to our iOS & Android React Native apps from the get-go
Work on integrating and collaborating with other crypto products to enrich the user experience
Implement features with the aim to provide long term stability of a highly scalable application
Developing clean and reusable react native components
Actively contributing new ideas and innovative UX/UI patterns
Play an active role in evolving our mobile engineering practices

Minimum Qualifications:
Strong expertise in React Native with 3-5+ years of professional experience building high-quality mobile apps
High-level experience with TypeScript
Have experience with best-in-class client-side data design and management, whether that’s experience with GraphQL/ Apollo or other such tools
Have a passion for good design and attention to detail. You’ll collaborate with designers but undoubtedly have the opportunity to express your own design sense if desired
Verbal fluency in English

Nice-to-Have Qualifications:
Demonstrable portfolio of released applications on the App Store or the Android
Passionate about crypto markets, blockchain technology, and decentralized applications
Knowledge/experience working on mobile financial applications/cryptocurrencies/wallets
Experience with Solana Web3 frameworks and technologies, especially on mobile
Experience working with a large codebase and detox or appium testing tools
Experience with responsive UI and debugging skills
Experience developing high-performance mobile app interfaces
Bachelors or Masters degree, preferably in Computer Science, Computer Engineering or other technical (STEM) field or an equivalent of experience

Benefits:
Competitive salary
Remote work in a dynamic, fast-growth Web3 startup
Flexible working hours
Opportunity to work with a diverse, global, and passionate team
Collaboration with leading projects and thought leaders in the crypto space
